Felienne/hedy
# [FEATURE] Enables users to set a preferred keyword language on their profile

**Description**
In this PR we implement the structure to enable users to set and change a preferred keyword language. With the implementation of multi-language keywords users are currently able to switch between their chosen language (if supported) and English through a language switcher on the code editor. With this PR users can set a fixed preferred command language, which is stored on their profile.

**Fix for**
This PR addresses, but does not fully fix #2008.

**How to test**
This is mostly a back-end structural PR and does not influence the user experience much. Navigate to the signup page and notice the new input field on `preferred keyword language`. Verify that this field is working as well as on the profile page where the user settings can be updated. If a language is chosen the language switcher on the front-end is no longer shown. 

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
